addons/website/views/snippets/snippets.xml

```diff
@@ -205,7 +205,7 @@
     </div>
 
     <div data-js="CarouselItem"
-         data-selector=".s_carousel .carousel-item">
+         data-selector=".carousel .carousel-item">
```

Comment:
Was like that before, was changed to s_carousel because of image gallery. I leave that commit out, move it to another PR and target quote carousel explicitely or exclude gallery snippet @Cocographique

Reply:
**PR** [46892](https://github.com/odoo/odoo/pull/46892)